state-chain/runtime/src/runtime_apis.rs

```diff
@@ -119,6 +121,50 @@ impl<BtcAddress> VaultSwapDetails<BtcAddress> {
 	}
 }
 
+pub mod validator_info_before_v7 {
+	use super::*;
+	#[derive(Encode, Decode, Eq, PartialEq, TypeInfo, Serialize, Deserialize)]
+	pub struct ValidatorInfo {
+		pub balance: AssetAmount,
+		pub bond: AssetAmount,
+		pub last_heartbeat: u32, // can *maybe* remove this - check with Andrew
+		pub reputation_points: i32,
+		pub keyholder_epochs: Vec<EpochIndex>,
+		pub is_current_authority: bool,
+		#[deprecated]
+		pub is_current_backup: bool,
+		pub is_qualified: bool,
+		pub is_online: bool,
+		pub is_bidding: bool,
+		pub bound_redeem_address: Option<EthereumAddress>,
+		pub apy_bp: Option<u32>, // APY for validator/back only. In Basis points.
+		pub restricted_balances: BTreeMap<EthereumAddress, AssetAmount>,
+		pub estimated_redeemable_balance: AssetAmount,
+	}
+}
+
+impl From<validator_info_before_v7::ValidatorInfo> for ValidatorInfo {
+	fn from(old: validator_info_before_v7::ValidatorInfo) -> Self {
+		ValidatorInfo {
+			balance: old.balance,
+			bond: old.bond,
+			last_heartbeat: old.last_heartbeat,
+			reputation_points: old.reputation_points,
+			keyholder_epochs: old.keyholder_epochs,
+			is_current_authority: old.is_current_authority,
+			is_current_backup: old.is_current_backup,
+			is_qualified: old.is_qualified,
+			is_online: old.is_online,
+			is_bidding: old.is_bidding,
+			bound_redeem_address: old.bound_redeem_address,
+			apy_bp: old.apy_bp,
+			restricted_balances: old.restricted_balances,
+			estimated_redeemable_balance: old.estimated_redeemable_balance,
+			operator: None,
+		}
+	}
+}
+
 #[derive(Encode, Decode, Eq, PartialEq, TypeInfo, Serialize, Deserialize)]
 pub struct ValidatorInfo {
```

Comment:
would be nice to have the `total_stake` or whatever which is the stake + delegation for this node but i suppose we can read it from the operator if it's a pain

Reply:
We can discuss this - I think another rpc is appropriate.